### ISSUE TITLE
Incorrect field name for timestamp fields

### ISSUE THREAD

**Issue by jbelkins** [awslabs/aws-sdk-swift#1121]
### Describe the bug

Internal reference: `SWIFT-171`

I'm getting this error message when calling `getMetricStatistics`, so I did a backtrace and found the code defect in the SDK:

1. https://github.com/awslabs/aws-sdk-swift/blob/main/Sources/Services/AWSCloudWatch/models/Models.swift#L4148
2. https://github.com/awslabs/aws-sdk-swift/blob/main/Sources/Services/AWSCloudWatch/models/Models.swift#L4172

Looks like AWS is expecting `EndTime` and `StartTime`, but you are encoding to lowercased key.

```
AWSCloudWatch.MissingRequiredParameterException(properties: AWSCloudWatch.MissingRequiredParameterException.Properties(message: nil), httpResponse: 
Status Code: http_status_400 
 x-amzn-RequestId: 18259d7f-896d-4382-93c4-a19891b97d96, 
Content-Type: text/xml, 
Connection: close, 
Content-Length: 290, 
Date: Tue, 12 Sep 2023 22:26:02 GMT, message: Optional("The parameter StartTime is required."), requestID: Optional("18259d7f-896d-4382-93c4-a19891b97d96"))
```

### Expected Behavior

`getMetricStatistics` operation should succeed

### Current Behavior

`getMetricStatistics` operation fails as described above

### Reproduction Steps

Perform operation as described above

### Possible Solution

Looks like capitalization of start time & end time field names do not match the model.
- In the model, they are `StartTime` and `EndTime`: https://github.com/awslabs/aws-sdk-swift/blob/main/codegen/sdk-codegen/aws-models/cloudwatch.2010-08-01.json#L2205-L2218
- In code, they are `startTime` and `endTime`: 1. https://github.com/awslabs/aws-sdk-swift/blob/main/Sources/Services/AWSCloudWatch/models/Models.swift#L4148-L4172

### AWS SWIFT SDK version used

Latest

### Compiler and Version used

Latest

### Operating System and version

Latest

**Comment by jbelkins** [comment]
Check this issue on all AWS protocols

**Comment by jbelkins** [comment]
A fix for this issue is ready and now in review.

This issue apparently does not affect any AWS protocol other than awsQuery, but it does affect 16 total services, including CloudWatch (as reported above).

**Comment by jbelkins** [comment]
Released in AWS SDK for Swift 0.26.0
